examples/platforms/da15000/README.md

```diff
@@ -110,10 +110,20 @@ Board will indicate state of device according to LED blink speed.
    16 bytes from fdde:ad00:beef:0:92f5:9844:67ad:a0c9: icmp_seq=2 hlim=64 time=359ms
    ```
 
+## Remarks
+* Validation
+
+   The DA15000 example has been validated by Dialog Semiconductor with commit #999 included.
```

Comment:
Prefer to reference the actual git commit ID (rather than the GitHub PR ID).  In this case, it should be 0bc2efe.